src/main/diagnostics/rdBinInShell.ts

```diff
@@ -67,7 +72,8 @@ class RDBinInShellPath implements DiagnosticsChecker {
   }
 }
 
-const RDBinInBash = new RDBinInShellPath('RD_BIN_IN_BASH_PATH', 'bash', '-i', '-c', 'echo $PATH');
-const RDBinInZsh = new RDBinInShellPath('RD_BIN_IN_ZSH_PATH', 'zsh', '-i', '-c', 'echo $PATH');
+// Use `bash -l` because `bash -i` causes RD to suspend
+const RDBinInBash = new RDBinInShellPath('RD_BIN_IN_BASH_PATH', 'bash', '-l', '-c', `printf "\n${ pathOutputDelimiter }%s\n" "$PATH"`);
+const RDBinInZsh = new RDBinInShellPath('RD_BIN_IN_ZSH_PATH', 'zsh', '-i', '-c', `printf "\n${ pathOutputDelimiter }%s\n" "$PATH"`);
```

Comment:
Would it make sense to add the `printf…` in the constructor, instead of here?

Reply:
Sure. If we support a shell that for whatever reason needs a different command, we can revisit this.